layouts/index.html

```diff
@@ -9,17 +9,37 @@
 
 
 
-<section class="section section-blue section-chevron_alt section-first"
-    id="main">
+<!-- <section class="section section-blue section-chevron_alt section-first"
+    id="main"> -->
+<section class="section-cream section-first" id="main">
   <div class="usa-grid section-bg-parent">
-    <div class="usa-width-whole billboard  section-bg">
-      <h1 class="billboard-display_text lh-hed">A Platform as a Service for government teams</h1>
-      <h2 class="billboard-second_text">Focus on the services you need to build — cloud.gov makes it faster, simpler, and more secure.</h2>
+    <div class="usa-width-two-thirds section-lede">
+      <!-- <h1 class="billboard-display_text lh-hed">A Platform as a Service for government teams</h1> -->
+      <p class="subhead-rule">A Platform as a Service for government teams</p>
+      <h1 class="billboard-second_text">
+        Focus on the services you need to build — cloud.gov makes it faster, simpler, and more secure. <br /><br/>
+        Read more about the <a href="/overview/overview/what-is-cloudgov/">product</a>, <a href="/overview/pricing/rates/">packages and rates</a>, and <a ref="/overview/technology/responsibilities/">security and compliance</a>.
+      </h1>
+    </div>
+    <div class="usa-width-one-third section-sidebar">
+      <p class="subhead-rule">News</p>
+      <h1>
+        <a href="https://www.digitalgov.gov/event/hands-on-workshop-with-cloud-gov/">Register now for hands-on cloud.gov workshop on Sep. 28</a>
+      </h1>
+      <p>
+        Learn how building, delivering and securing web applications in cloud.gov differs from typical IaaS. In this hands-on workshop, you’ll launch a live web app on cloud.gov.
+      </p>
+      <p>
+        Join onsite at 1800 F Street, Washington, DC, or remotely via YouTube Live.
+      </p>
+      <p>
+        <a class="button">Registration details</a>
```

Comment:
Do we need a url here? https://www.digitalgov.gov/event/hands-on-workshop-with-cloud-gov/